ingestion/src/metadata/interfaces/sqa_interface.py

```diff
@@ -275,6 +275,8 @@ def compute_metrics_in_thread(
         if column is not None:
             column = column.name
 
+        session.close()
```

Comment:
do we need to wrap it around try/except here. What if session.close() throws a exception

Reply:
@harshach what kind of exception are you thinking of? We open the session just above so that instance will have a close() attribute.

Reply:
maybe we can check if our sqlalchemy support something like

```python
with Session() as session:
    ...
```

and use use the internal context management

Reply:
we're able to do that on the Airflow REST APIs on 2.3.3

Reply:
@pmbrull yes should be much cleaner. Just pushed a change.